ncl/ncl_entries/day_of_week.ipynb

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
<p>I'm going to regret bringing this up, but do we want the grab and go sections for ncl entries to <em>replicate </em>ncl?</p><p> </p><p>Like, here, should we point out that to make it "equivalent" you have to shift the start day?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Maybe, if it's easy, but otherwise I think a section that discusses differences might be good enough.  </p><p> </p><p>Feel free to give thoughts on items like this Andy, but also no pressure.  This might be a little confusing as well without some context.</p>

Reply:
I'm going to drop my concern on this PR, I've added it to the discussion items for Monday.